### PR TITLE
Fixed clean install and instructions

### PR DESCRIPTION
**Pull request by igorline**
Hey, @palevoo ! Can you verify that with current changes you are able to do a fresh setup?

### CODE REVIEW

**Comment by igorline** [comment]
@palevoo I have just pushed 3 more commits and verified that I'm able to fresh install on a new machine. Can you check if you are able to start it up?

I'm missing:
- [ ] instructions to start backend
- [ ] instructions to start bot
- [ ] instructions to run tests

File: packages/kyodo-contracts/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.7.0",
     "babel-register": "^6.26.0",
-    "ganache-cli": "^6.1.3",
+    "ganache-cli": "6.1.8",
```
Comment (davidmurdoch) [inline packages/kyodo-contracts/package.json]: Fixed in [ganache-cli@6.2.2](https://github.com/trufflesuite/ganache-cli/releases/tag/v6.2.2). Let me know if you have any more issues.

**Comment by igorline** [comment]
Ok, should be done by now.
Repo is missing instructions to start kyodo bot, will be fixed in #183.

I'm adding fixed outdated tests for smart contracts now and will fire an issue to setup a circleci integration for CI/CD.

File: docs/README.md [hunk 2]
```diff
@@ -2,8 +2,82 @@
...
+# Create initial distribution of tokens
+# Change `deploy_parameters.json` to reflect the initial distribution you want or leave the accounts section empty.
+cp migrations/deploy_parameters.example.json migrations/deploy_parameters.json
+```
+
+
+### Setup contracts
+
+```bash
+cd packages/kyodo-contracts
```
Comment (jsidorenko) [inline docs/README.md]: What do you think about this starting [approach](https://github.com/JoinColony/colonyStarter/blob/master/package.json)? 
It eliminates the need to navigate to the `packages/kyodo-contracts` folder and allows the developer to launch all the scripts from the project root

**Comment by igorline** [comment]
Related ticket #127

**Pull request merged**